Dockerfile

```diff
@@ -1,41 +1,28 @@
-ARG py_env_path=/env
-ARG V_BASE=3.3.0
-
-FROM opendatacube/geobase-builder:${V_BASE} as env_builder
-ENV LC_ALL=C.UTF-8
-
-# Install our Python requirements
-COPY docker/requirements.txt docker/version.txt docker/constraints.txt /conf/
-
-RUN cat /conf/version.txt && \
-  env-build-tool new /conf/requirements.txt /conf/constraints.txt ${py_env_path}
-
-# Install the crop mask tools
-ADD production/cm_tools /tmp/cm_tools
-RUN /env/bin/pip install \
-  --extra-index-url="https://packages.dea.ga.gov.au" /tmp/cm_tools && \
-  rm -rf /tmp/cm_tools
-
-# Below is the actual image that does the running
-FROM opendatacube/geobase-runner:${V_BASE}
-ARG py_env_path=/env
+FROM osgeo/gdal:ubuntu-small-3.4.1
 
 ENV DEBIAN_FRONTEND=noninteractive \
-    PATH="${py_env_path}/bin:${PATH}" \
     LC_ALL=C.UTF-8 \
     LANG=C.UTF-8
 
+# Install system tools
 RUN apt-get update \
-    && apt-get install software-properties-common -y \
-    && apt-get upgrade -y
-    
-# Add in the dask configuration
-COPY docker/distributed.yaml /etc/dask/distributed.yaml
+    && apt-get install software-properties-common -y
+
 ADD docker/apt-run.txt /tmp/
 RUN apt-get update \
     && sed 's/#.*//' /tmp/apt-run.txt | xargs apt-get install -y \
     && rm -rf /var/lib/apt/lists/*
 
+# Install our Python requirements
+RUN mkdir -p /conf
+COPY docker/requirements.txt docker/version.txt docker/constraints.txt /conf/
+
+RUN pip install --no-cache-dir -r /conf/requirements.txt
```

Comment:
We need. to use the constraints file here by including `-c /conf/constraints.txt`, if we're making it. Otherwise if we don't actually need it, we should remove it.

Another way to do this is to have a text file `requirements.in` and use that with `pip-compile` to make `requirements.txt`.